docs/data/data-grid/clipboard/ClipboardImport.js

```diff
@@ -0,0 +1,249 @@
+    desk: 'D-5285',
+    commodity: 'Cocoa',
+    traderName: 'Alberta Greene',
+    traderEmail: 'tudik@sivni.ch',
+    quantity: 41917,
+  },
+  {
+    id: 'eaf40294-b985-58ee-901d-4b454a86b010',
+    desk: 'D-9333',
+    commodity: 'Soybeans',
+    traderName: 'Nancy Bush',
+    traderEmail: 'witac@defjawva.ec',
+    quantity: 68330,
+  },
+  {
+    id: 'd152e01c-fac0-5666-8fe4-e409e370f41b',
+    desk: 'D-5129',
+    commodity: 'Soybean Meal',
+    traderName: 'Earl Burns',
+    traderEmail: 'bevloha@curokvas.gh',
+    quantity: 47471,
+  },
+  {
+    id: 'e0e2b8ac-0091-5c5b-89eb-cd30cc538b93',
+    desk: 'D-1635',
+    commodity: 'Sugar No.14',
+    traderName: 'Landon Steele',
+    traderEmail: 'vonvuz@rufer.kr',
+    quantity: 26542,
+  },
+];
+
+const useSessionStorageData = () => {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 0,
+    maxColumns: 6,
+    editable: true,
+  });
+
+  const [rows] = React.useState(() => {
+    try {
+      const lsData = sessionStorage.getItem('clipboardImportRows');
+      if (lsData) {
+        const parsedData = JSON.parse(lsData);
+        if (Array.isArray(parsedData)) {
+          return parsedData;
+        }
+      }
+    } catch (error) {
+      // session storage is not available
+    }
+    return initialRows;
+  });
+
+  const rowsRef = React.useRef([...rows]);
+
+  const updateRow = React.useCallback((newRow) => {
+    const index = rowsRef.current.findIndex((row) => row.id === newRow.id);
+    rowsRef.current[index] = newRow;
+    sessionStorage.setItem('clipboardImportRows', JSON.stringify(rowsRef.current));
+  }, []);
+
+  return {
+    data: { columns: data.columns, rows },
+    updateRow,
+  };
+};
+
+export default function ClipboardImport() {
+  const [rowSelection, setRowSelection] = React.useState(false);
+  const { data, updateRow } = useSessionStorageData();
+  const [loading, setLoading] = React.useState(false);
+
+  const apiRef = useGridApiRef();
+
+  React.useEffect(() => {
+    return apiRef.current.subscribeEvent('clipboardPasteStart', () => {
+      setLoading(true);
+    });
+  }, [apiRef]);
+
+  React.useEffect(() => {
+    return apiRef.current.subscribeEvent('clipboardPasteEnd', () => {
+      setLoading(false);
+    });
+  }, [apiRef]);
+
+  return (
+    <div style={{ width: '100%' }}>
+      <Button sx={{ mb: 2 }} onClick={() => setRowSelection(!rowSelection)}>
+        Toggle row selection
+      </Button>
+      <div style={{ height: 400 }}>
+        <DataGridPremium
+          apiRef={apiRef}
+          loading={loading}
+          rowSelection={rowSelection}
+          checkboxSelection={rowSelection}
+          unstable_cellSelection
+          {...data}
+          // getRowId={(row) => row.rowId}
+          processRowUpdate={(newRow) => {
+            updateRow(newRow);
+            return newRow;
+          }}
+          unstable_enableClipboardPaste
+          experimentalFeatures={{ ignoreValueFormatterDuringExport: true }}
```

Comment:
This flag adds behavior that was initially proposed in https://github.com/mui/mui-x/pull/7809
Without it, I don't see how we could support copy/paste between cells.
Take a numeric column for example: https://github.com/mui/mui-x/blob/2da594f20c4dd022a5105b8db421c1d78511e923/packages/grid/x-data-grid/src/colDef/gridNumericColDef.ts#L14

We format numbers using `.toLocaleString`, which could give different values depending on the locale. `1000.5` could be formatted as:
- `1,000.5`
- `1000.5`
- `1000,5`
- `1 000,5`
If we continue copying formatted values, it's almost impossible to parse the number properly.
I'm open to discussion on this topic though :)